components/NavigationBar/examples/NavigationBarTypicalUseExample.swift

```diff
@@ -29,10 +29,12 @@ open class NavigationBarTypicalUseSwiftExample: NavigationBarTypicalUseExample {
     navBar = MDCNavigationBar()
```

Comment:
maybe not for this PR, but would be nice to get rid of all the forced unwrapping below.
 
```
guard let navBar = navBar else {
  return
}
```

Reply:
Agreed.

https://www.pivotaltracker.com/story/show/156619606